nucypher/characters/lawful.py

```diff
@@ -71,26 +72,36 @@
 from nucypher.network.server import ProxyRESTServer, TLSHostingPower, make_rest_app
 
 
-class Alice(Character, PolicyAuthor):
+class Alice(Character, BlockchainPolicyAuthor):
 
     banner = ALICE_BANNER
     _controller_class = AliceJSONController
     _default_crypto_powerups = [SigningPower, DecryptingPower, DelegatingPower]
 
     def __init__(self,
+
+                 # Mode
+                 is_me: bool = True,
+                 federated_only: bool = False,
```

Comment:
Weird that you can set this to `True` and it inherits from `BlockchainPolicyAuthor`.

Reply:
Yes, how does this work in federated mode?